CRM/Campaign/Form/Petition.php

```diff
@@ -126,6 +126,10 @@ public function preProcess() {
    *   array of default values
    */
   public function setDefaultValues() {
+    if ($this->_cdType) {
+      return CRM_Custom_Form_CustomData::setDefaultValues($this);
```

Comment:
Didn't we actually get rid of this from all the other forms? In 4.7 we refactored away the concept of custom form as a distributed form